tools/functional-tester/etcd-runner/command/election_command.go

```diff
@@ -65,8 +68,8 @@ func runElectionFunc(cmd *cobra.Command, args []string) {
 				break
 			}
 		}
-		e := concurrency.NewElection(s, "electors")
 
+		e := concurrency.NewElection(s, elector)
```

Comment:
why does this need to change? is it because it's possible there's a conflict? If so, how does having a random string solve the conflict, instead of just making it rare?